apphub/tabular/dnn_breast_cancer/dnn_breast_cancer.py

```diff
@@ -40,7 +42,8 @@ def create_dnn():
 
 
 def get_estimator(epochs=50, batch_size=32, steps_per_epoch=None, validation_steps=None, model_dir=tempfile.mkdtemp()):
-    (x_train, y_train), (x_eval, y_eval) = tf.keras.datasets.boston_housing.load_data()
+    (X, y) = load_breast_cancer(True)
```

Comment:
the original host of this dataset mentions that the included an ID column in their CSV. does the sklearn version get rid of that automatically?

Reply:
nvm, looks like they handled it